lib/p12.ml

```diff
@@ -431,13 +431,20 @@ let verify password (data, ((algorithm, digest), salt, iterations)) =
   end else
     Error (`Msg "invalid signature")
 
-let create ?(mac = `SHA256) ?(algorithm = `AES256_CBC) ?(iterations = 2048) password certificates private_key =
+let create ?(mac = `SHA256) ?(algorithm = `AES256_CBC) ?(iterations = 2048) ?local_key_id password certificates private_key =
```

Comment:
should we allow arbitrary `attributes`, and not only `local_key_id`?

so, would ` let create ?(mac = `SHA256) ?(algorithm = `AES256_CBC) ?(iterations = 2048) ?attributes password certificates private_key =` work fine? Hmm, but then we'd need to expose the `Registry.PKCS9`... I'm fine with the `local_key_id` as well.